### PR TITLE
Start screen

### PR DESCRIPTION
**Pull request by JessWhosBack**
Issues: #4 #5 
Created the landing page for the web app as well as the basics of the lobby page. From this lobby user can select a 'Standard' or 'Custom' game mode and this creates a new entry in the Game table in the database and sends the user to the game page.

### CODE REVIEW

File: mainRoutes.js [hunk 1]
```diff
@@ -1,10 +1,49 @@
...
+  res.sendFile(path.join(__dirname, 'views', 'home.html'))
 })
+
 mainRouter.get('/about', function (req, res) {
-    res.sendFile(path.join(__dirname, 'views', 'about.html'))
+  res.sendFile(path.join(__dirname, 'views', 'about.html'))
 })
-module.exports = mainRouter
\ No newline at end of file
+mainRouter.get('/form', function (req, res) {
```
Comment (Ray-R4y) [inline mainRoutes.js:16]: Good use of using routes as the communication channel between the client side and the server side

**Review by Ray-R4y — APPROVED** [review]
**Design**
Interaction between the server and client is good, as it uses routes to communicate between the server and the database, and the route has no logic in there.

**Functionality**
The code works well as the design is simple and designed to have minimum errors. Code works as expected as it is simple button presses.

**Complexity**
Code is simple to understand and there are no big functions. every function does what it needs to do

**Tests**
Testing is done to check the basic functionalities

**Naming**
The developer has picked good names for everything

**Comments**
The developer lack comments, the code is simple enough to understand but comments in future would be recommended.

**Style**
The style follows the google javascript style guide

**Consistency**
The code is consistent with the style guide

**Documentation**
All Documentations exist

**Exceptions**
most exceptions are handled apart from a few cases that need to be added.

**Good things** 
The design is good and the code is not complex

File: package.json [hunk 2]
```diff
@@ -15,6 +14,17 @@
...
+    "jest": "^28.1.0",
+    "mssql": "^8.1.1",
+    "mysql": "^2.18.1",
+    "node": "^17.7.2",
+    "tedious": "^14.5.0"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "keywords": []
```
[the remark this answers is not in the document]
Comment (JessWhosBack) [reply package.json:29]: Noted! Thanks for noticing. Will add before end of next sprint.

File: services/lobby.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+async function createGame (gameMode, newID) {
+  return new Promise((resolve, reject) =>{
+    const sqlCode = `INSERT INTO dbo.Game(GameID, GameType) VALUES ('${newID}', '${gameMode}');`
+    get('default').then((pool) =>
+        pool.request().query(sqlCode).then(
+          (result) => {
+            resolve(result)
+          })//.catch(console.error)
```
[the remark this answers is not in the document]
Comment (JessWhosBack) [reply services/lobby.js:24]: Yes! I will work on this immediately.

**Pull request merged**